src/releng_docs/services_shipit.rst

```diff
@@ -57,3 +60,44 @@ It does the following tasks on every run:
 
 
 .. _libmozdata: https://github.com/mozilla/libmozdata/
```

Comment:
Yes this library from Release Management is used to do the patch analysis in shipit_bot_uplift.
You can see the link used on https://docs.mozilla-releng.net/services_shipit.html#src-shipit-bot-uplift